vulcan/_api.py

```diff
@@ -23,6 +22,15 @@
     urlencode,
     uuid,
 )
+from .exceptions import (
```

Comment:
Should be in `_exceptions` like the rest of the project. Then these classes can be exported in `__init__.py`.

Reply:
imo `exceptions` loks better when we are importing exceptions in code

Reply:
If so, then a package named `exceptions`, still having imports in `__init__` though. It looks weird if a single file in the entire project is named differently.